### DISCUSSION TITLE
Uyuni proxy not working as expected

### DISCUSSION THREAD

**Discussion by re-verse** [uyuni-project/uyuni#7461] [Q&A]
I've set up an uyuni proxy on a different network than my uyuni server, and while it seems to be able to reach servers on the new network for bootstrapping, it seems it can't proxy files across from the server to the client.

The Standard Error I see is this:
`Error: Unable to download https://uyuni-proxy:443/pub/repositories/res/7/bootstrap/venv-enabled-x86_64.txt file!`
Looking further, I see this when i try to grab the file manually from the client for testing purposes:
` https://uyuni-proxy/pub/repositories/res/7/bootstrap/venv-enabled-x86_64.txt
Resolving uyuni-proxy (uyuni-proxy)... 172.29.1.122
Connecting to uyuni-proxy (uyuni-proxy)|172.29.1.122|:443... connected.
OpenSSL: error:140770FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol
Unable to establish SSL connection.`

It seems like the proxy is unable to serve SSL, and if i switch it to http for the same url, I get a 404 instead.

**Comment by dvosburg** [comment]
Is 'uyuni-proxy' in the CNAME aliasl list for your proxy's certificate?  Usually the certificate references only the FQDN by default, not the short name.

**Comment by re-verse** [comment]
It was not. I'm going to look at that now, and will report back.

**Comment by re-verse** [comment]
Thanks for that, it appears this was the problem. Once I re-ran the configure.proxy script paying special attention to add the cname it worked right away. I appreciate the guidance.
